ifrs17/Constants/Enums.ipynb

```diff
@@ -277,7 +277,15 @@
         {
             "cell_type": "code",
             "source": [
-                "public enum DataType { Optional, Mandatory, Calculated, CalculatedTelescopic }"
+                "[Flags]",
+                "\npublic enum DataType {Optional = 1, Mandatory = 2, Calculated = 4, CalculatedTelescopic = 8}",
+                "\n//Optional + Mandatory = 3",
```

Comment:
Optional + Mandatory? Does it make sense? Aren't this mutually exclusive?
If not, naming is weird. And also, if this makes sense, you miss some combinations (like 7)

Reply:
Maybe I would write a comment after each base option to know what its meant:

```
Enum DataType {

     Optional = 1, // This is for bla bla
     
     Mandatory = 2  // Use this in case of tralala

```

Reply:
I added some description. But as we discussed we might come back to these very soon.